nubia/internal/commands/help.py

```diff
@@ -55,7 +52,7 @@ def run_interactive(self, _0, args, _2):
             commands = {
                 cmd_name: cmd
                 for cmd in self.registry.get_all_commands()
-                for cmd_name in cmd.get_command_names()
+                for cmd_name in await try_await(cmd.get_command_names())
```

Comment:
I don't think we need `try_await` here as `get_command_names` is not `async` anywhere. It _could_ be made async, @AhmedSoliman what do you think? Should we basically allow all of the methods of `Command` to be async or sync?